airflow/api_fastapi/core_api/routes/public/monitor.py

```diff
@@ -27,4 +27,4 @@
 @monitor_router.get("/health")
 def get_health() -> HealthInfoSchema:
     airflow_health_status = get_airflow_health()
-    return HealthInfoSchema.model_validate(airflow_health_status)
+    return HealthInfoSchema.model_validate(airflow_health_status, from_attributes=True)
```

Comment:
```suggestion
    return airflow_health_status
```

Reply:
I think we need to cast it to  `HealthInfoSchema`

Reply:
mypy giving me below error

```
airflow/api_fastapi/core_api/routes/public/monitor.py:30: error: Incompatible
return value type (got "dict[str, Any]", expected "HealthInfoSchema")
[return-value]
        return airflow_health_status
               ^~~~~~~~~~~~~~~~~~~~~
Found 1 error in 1 file (checked 1193 source files)
```

Reply:
`from_attributes` should be removed.

Because it is now in the model configuration. This is done automatically.